nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -259,11 +277,24 @@ BtNavigator::navigateToPose()
       action_server_->publish_feedback(feedback_msg);
     };
 
+  auto bt_xml_filename = action_server_->get_current_goal()->behavior_tree;
+
+  // Empty id in request is default for backward compatibility
+  if (bt_xml_filename == "") {
```

Comment:
This can be put on one line:

bt_xml_filename = bt_xml_filename == "" ? default_bt_xml_filename_ : bt_xml_filename;